third_party/3/packaging/tags.pyi

```diff
@@ -0,0 +1,36 @@
+from typing import FrozenSet, Iterable, Iterator, Optional, Sequence, Tuple
+
+PythonVersion = Sequence[int]
+MacVersion = Tuple[int, int]
+
+class Tag:
+    def __init__(self, interpreter: str, abi: str, platform: str) -> None: ...
+    @property
+    def interpreter(self) -> str: ...
+    @property
+    def abi(self) -> str: ...
+    @property
+    def platform(self) -> str: ...
+    def __eq__(self, other: object) -> bool: ...
+    def __hash__(self) -> int: ...
+
+def parse_tag(tag: str) -> FrozenSet[Tag]: ...
+def cpython_tags(
+    python_version: Optional[PythonVersion] = ...,
+    abis: Optional[Iterable[str]] = ...,
+    platforms: Optional[Iterable[str]] = ...,
+    **kwargs: bool,
```

Comment:
This and the other **kwargs function actually take a keyword-only parameter `warn: bool = ...`; the **kwargs are a convoluted mechanism to support keyword-only arguments in Python 2. Keyword-only syntax works in stubs regardless of Python version, so we should just write `*, warn: bool = ...`.